test/Microsoft.ML.Tests/Scenarios/HousePricePredictionTests.cs

```diff
@@ -49,6 +52,64 @@ public partial class Top5Scenarios : BaseTestClass
             Assert.InRange(prediction.Price, 260_000, 330_000);
         }
 
+        [Fact(Skip = "Missing data set. See https://github.com/dotnet/machinelearning/issues/3")]
+        public void TrainAndPredictHousePriceModelTest()
```

Comment:
@KrzysztofCwalina specifically asked that we have 1 scenario test per file.  Would it be OK to keep the tests separate?

Reply:
# Resolved